### PR TITLE
JC getRoute getStockData [JC-1]

### PR DESCRIPTION
**Pull request by jorgecan101**
* **Please check if the PR fulfills these requirements**
- [x] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
Adds getStockData to be used as a get request called via postman or other HTTP request service

* **What is the current behavior?What does the code do before you made the new changes?** 
No routes existed prior to this

* **What is the new behavior?**
Allows user to view data for a stock when called

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
Only to note that this route now exists too. Nothing new of note will be needed to run this.

* **Other information**:
n/a

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const express = require("express")
+const app = express()
+const axios = require('axios')
+app.use(express.json())
+
+//Route to get a stock from the database. Calling https://codubee-projects-api.herokuapp.com/stocks/getStockData
```
Comment (Codubee) [inline index.js]: This comment may be a bit unnecessary here. Doesnt tell me much about the code that I cant figure out by reading the code.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+const axios = require('axios')
+app.use(express.json())
+
+//Route to get a stock from the database. Calling https://codubee-projects-api.herokuapp.com/stocks/getStockData
+app.get('/getStockData', function(postmanReq, postmanRes) {
+    console.log("Getting Data for a Stock...")
+    axios.get('https://codubee-projects-api.herokuapp.com/stocks/getStockData')
+    .then((response) => {
+        console.log(response.data)
+        postmanRes.status(200).json({"message":response.data})
```
Comment (Codubee) [inline index.js]: Here the key is message. Would a key of message be good, or is a key of data better?

**Review by Codubee — CHANGES_REQUESTED** [review]
Great job! Just left a few comments.

File: index.js (hunk 1 above)
Comment (arsoljon) [reply index.js]: Agree with @Codubee

File: index.js (hunk 2 above)
Comment (arsoljon) [reply index.js]: Agree with @Codubee

**Comment by jorgecan101** [comment]
Made changes based on feedback

**Review by arsoljon — DISMISSED** [review]
Looks good to me.

**Review by Codubee — DISMISSED** [review]
LGTM!

**Comment by jorgecan101** [comment]
fixed merge conflicts

**Review by arsoljon — APPROVED** [review]
Looks good.

**Review by Codubee — APPROVED** [review]
LGTM!

**Pull request merged**